docs/source/borealis_data.rst

```diff
@@ -18,3 +24,22 @@ The following data filetypes can be generated by Borealis in hdf5 format. The st
     rawacf
 
 Additionally, debugging file formats such as rawrf (non-filtered high bandwidth data) can be generated upon request. 
+
+Reading Data
+-------------
+
+To read the files in python, we recommend using `PyTables <https://www.pytables.org/>`_ or `deepdish <https://deepdish.readthedocs.io/en/latest/index.html>`_ packages.
+
+
+*************************
+Data Storage and Deletion
+*************************
+
+Borealis file sizes can add up quickly to fill all available hard drive space, especially if antennas_iq and/or bfiq data types are being generated. However, it is convenient and recommended to keep a backlog of lower level data products such as antennas_iq for a period of time. These files are useful for debugging hardware issues and reproducing RAWACF files.
+
+In order to prevent system failure due to hard drives filling up, a method for deleting the oldest data files is employed. This is referred to as `rotating` the files.
```

Comment:
I would move this sentence under File Rotation and edit it to clarify this is a separate thing we use at SuperDARN Canada that might be useful to know but isn't anything attached to the Borealis package. 
```suggestion
In order to prevent system failure due to hard drives filling up, a method for deleting the oldest data files is employed for SuperDARN Canada radars. This is referred to as `rotating` the files.
```